src/vosk_api.h

```diff
@@ -212,6 +213,9 @@ void vosk_recognizer_free(VoskRecognizer *recognizer);
  */
 void vosk_set_log_level(int log_level);
 
+void vosk_gpu_init();
```

Comment:
Could you add doxygen comments like other methods please?

Reply:
Will add. But there's a quick question.

According to [Kaldi sources](https://github.com/kaldi-asr/kaldi/blob/master/src/cudamatrix/cu-device.h#L82-L91), `CuDevice::Instantiate()` returns an instance of `CuDevice` for a current thread. I understand why do we need to call it within `vosk_gpu_init()`, as it selects gpu id and allows multithreading.

But what I can't get is why do we need a separate `vosk_gpu_instantiate()` method, which just calls a `CuDevice` initializer, but nothing more? Isn't it enough just to call `vosk_gpu_init()` which already encapsulates `CuDevice::Instantiate()`? These 2 methods seem very confusing at the moment.

Reply:
Added docs + renamed `vosk_gpu_instantiate`.